fml/raster_thread_merger_unittests.cc

```diff
@@ -301,3 +304,93 @@ TEST(RasterThreadMerger, HandleTaskQueuesAreTheSame) {
   term1.Signal();
   thread1.join();
 }
+
+TEST(RasterThreadMerger, FirstTaskMergesAndUnmergesThreads) {
+  fml::MessageLoop* loop1 = nullptr;
+  fml::AutoResetWaitableEvent latch1;
+  std::thread thread1([&loop1, &latch1]() {
+    fml::MessageLoop::EnsureInitializedForCurrentThread();
+    loop1 = &fml::MessageLoop::GetCurrent();
+    loop1->GetTaskRunner()->PostTask([&]() { latch1.Signal(); });
+    loop1->Run();
+  });
+
+  fml::MessageLoop* loop2 = nullptr;
+  fml::AutoResetWaitableEvent latch2;
+  std::thread thread2([&loop2, &latch2]() {
+    fml::MessageLoop::EnsureInitializedForCurrentThread();
+    loop2 = &fml::MessageLoop::GetCurrent();
+    loop2->GetTaskRunner()->PostTask([&]() { latch2.Signal(); });
+    loop2->Run();
+  });
+
+  latch1.Wait();
+  latch2.Wait();
+
+  fml::TaskQueueId qid1 = loop1->GetTaskRunner()->GetTaskQueueId();
+  fml::TaskQueueId qid2 = loop2->GetTaskRunner()->GetTaskQueueId();
+  const auto raster_thread_merger_ =
+      fml::MakeRefCounted<fml::RasterThreadMerger>(qid1, qid2);
```

Comment:
minor nit: maybe these can be named raster and platform queues, so it's easier to follow?

Reply:
done